llarp/router/router.cpp

```diff
@@ -110,11 +110,18 @@ namespace llarp
   util::StatusObject
   Router::ExtractStatus() const
   {
-    return util::StatusObject{
-        {"dht", _dht->impl->ExtractStatus()},
-        {"services", _hiddenServiceContext.ExtractStatus()},
-        {"exit", _exitContext.ExtractStatus()},
-        {"links", _linkManager.ExtractStatus()}};
+    if(_running)
+    {
+      return util::StatusObject{
+          {"dht", _dht->impl->ExtractStatus()},
+          {"services", _hiddenServiceContext.ExtractStatus()},
+          {"exit", _exitContext.ExtractStatus()},
+          {"links", _linkManager.ExtractStatus()}};
+    }
+    else
+    {
+      return util::StatusObject{{"notRunning", true}};
+    }
```

Comment:
this should set `running` to `true` if running and return only an object with `running` as `false` in the case it's not.